src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/updateCVECandidates.hpp

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
Should this class be part of `storeXXXModel`?

Reply:
I'm taking the same approach as
[Save vulnerability description in rocksDB database](https://github.com/wazuh/wazuh/pull/19440#top)
#19440

I thought that another class would orchestrate the required methods, but if this isn't the case, then we can sync the PRs and change it.

Reply:
I see, but I don't remember that detail.
Thank you.